x-pack/plugins/enterprise_search/server/routes/app_search/result_settings.ts

```diff
@@ -53,7 +53,7 @@ export function registerResultSettingsRoutes({
       },
     }),
     enterpriseSearchRequestHandler.createRequest({
-      path: '/as/engines/:engineName/sample_response_search',
+      path: '/as/engines/:engineName/dashboard_search',
```

Comment:
Should we be opting to delete the `/sample_response_search` server route entirely and simply having the frontend hit the new `/search` API endpoint?

Reply:
Oh shoot, I meant to do that, one sec.

Reply:
Ah, I had it locally but forgot to save the file. Pushing it up now...

Reply:
11eefc6d31ab04264e11220479526793d09ebc09

Reply:
Nice!!